app/uk/gov/hmrc/gform/views/form/addToList.scala.html

```diff
@@ -79,8 +79,9 @@
 
     @pageLevelError.render
 
-    @hmrcPageHeading(PageHeading(text = heading))
```

Comment:
If we don't use `hmrcPageHeading` value anymore, let's clean it up and remove import

Reply:
Good point, done